<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClasificadorClass</class>
 <widget class="QMainWindow" name="CClasificadorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CClasificador</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CQtImageManager" name="video" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ssButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>771</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>771</width>
      <height>68</height>
     </rect>
    </property>
   </widget>
   <widget class="CQtImageManager" name="cuadro" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>230</y>
      <width>250</width>
      <height>298</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ssButton_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>540</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver leyenda</string>
    </property>
   </widget>
   <widget class="CQtImageManager" name="cuadro_2" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>250</width>
      <height>298</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1448</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQtImageManager</class>
   <extends>QWidget</extends>
   <header>qtimagemanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Clasificador.qrc"/>
 </resources>
 <connections/>
</ui>
